src/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+export {
+  getTemplateLocation,
+  rootDirectory,
+  pathWhereScriptIsRunning,
+  DEFAULT_TEMPLATE_PATH
+} from "./getTemplateLocation";
```

Comment:
can this use `export * from "./getTemplateLocation"`?

Reply:
Sure

Reply:
Done!